### PR TITLE
Rails 8

### PR DESCRIPTION
**Pull request by chao-xian**
### What problem does this pull request solve?

Trello card: 

### Things to consider when reviewing

- Does it work when run on your machine?
- Is it clear what the code is doing?
- Do the commit messages explain why the changes were made?
- Are there all the unit tests needed?
- Has all relevant documentation been updated?

### CODE REVIEW

File: bin/dev
Comment (DavidBiddle) [inline bin/dev:1]: I don't think we want this change, at least as it's currently written - our existing bin/dev is set up to run the Rails server and the vite dev task at the same time (see the [Procfile](https://github.com/alphagov/forms-product-page/blob/main/Procfile.dev)) whereas this new version only runs the Rails server.

Comment (DavidBiddle) [reply bin/dev:1]: (running the Vite dev task at the same time as the Rails server gives us hot module reloading when editing JS/CSS)

Comment (chao-xian) [reply bin/dev:1]: ah good spot thank you

File: config/initializers/filter_parameter_logging.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 # Use this to limit dissemination of sensitive information.
 # See the ActiveSupport::ParameterFilter documentation for supported notations and behaviors.
 Rails.application.config.filter_parameters += %i[
-  passw secret token _key crypt salt certificate otp ssn
+  passw secret token _key crypt salt certificate otp ssn cvv cvc
```
Comment (DavidBiddle) [inline config/initializers/filter_parameter_logging.rb:7]: Hopefully we won't be touching CVV/CVCs anyway but no harm in having it in here

**Review by DavidBiddle — APPROVED** [review]

**Pull request merged**